packages/next/src/generators/init/utils/react-axe.ts

```diff
@@ -0,0 +1,86 @@
+import {
+    addDependenciesToPackageJson,
+    ProjectConfiguration,
+    joinPathFragments,
+} from '@nx/devkit';
+import chalk from 'chalk';
+import { Project, SyntaxKind } from 'ts-morph';
+
+// the code that needs to be injected to _app.tsx file.
+const reactAxeConfigurationCode = `
+/**
+ * A dynamic import is used here to only load the react-axe library for a11y checks
+ * when it is not in production mode
+ * This ensures that it is only ran in local env.
+ */
+// @ts-ignore
+if (typeof window !== 'undefined' && process.env.NODE_ENV !== 'production' ) {
+    // eslint-disable-next-line global-require
+    const axe = require('@axe-core/react'); // eslint-disable-line @typescript-eslint/no-var-requires
+    // eslint-disable-next-line global-require
+    const React = require('react'); // eslint-disable-line @typescript-eslint/no-var-requires
+    // eslint-disable-next-line global-require
+    const ReactDOM = require('react-dom'); // eslint-disable-line @typescript-eslint/no-var-requires
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    axe(React, ReactDOM, 1000);
+}
+`;
+
+export function addReactAxeDependency(tree) {
+    return addDependenciesToPackageJson(
+        tree,
+        {},
+        {
+            '@axe-core/react': '^4.7.3',
```

Comment:
We're using a constants file in other generators for the version numbers `packages/rest-client/utils/versions.ts`.
Do we want to do the same here?

Reply:
good shout! added constant